### PR TITLE
Initial insight integration

### PR DESCRIPTION
**Pull request by zanitete**
* hide header&footer
* hide code cells by default
* circleci config

### CODE REVIEW

**Review by julienbachmann — COMMENTED** [review]
It looks good for me. 

But why do we not just use the forked version of nbviewer. So we decide when we want to update it with nbviewer changes.  Also doing this will avoid having to have a script that check that the file changes and also having a custom docker file that re-copy the changed files.

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      - run:
+          name: clone product repo
+          command: git clone git@github.com:nagra-insight/product.git ~/product
+      - setup_remote_docker
+      - deploy:
+          name: Build and push Docker image
+          command: |
+            if [ "${CIRCLE_BRANCH}" == "master" ]; then
+              docker login -e="." -u="nagra+circleci_bot" -p $DOCKER_QUAY_PWD quay.io
+              ~/product/build/circle-docker-build.sh . ni_nbviewer $CIRCLE_SHA1 Dockerfile.insight
```
Comment (sgenoud) [inline .circleci/config.yml]: don't forget to create the repo in quay

**Review by sgenoud — APPROVED** [review]

**Comment by zanitete** [comment]
@julienbachmann I used a custom Dockerfile to have a clear separation between our changes and the upstream. But indeed I see your point and your solution reduces complexity and we better control which upstream changes integrate and when. I'll go for it, thanks

**Review by julienbachmann — APPROVED** [review]
👍

**Pull request merged**